src/com/mambu/apisdk/util/MambuEntity.java

```diff
@@ -0,0 +1,40 @@
+package com.mambu.apisdk.util;
+
+import com.mambu.clients.shared.model.Client;
+import com.mambu.clients.shared.model.Group;
+import com.mambu.core.shared.model.Comment;
+import com.mambu.core.shared.model.User;
+import com.mambu.loans.shared.model.LoanAccount;
+import com.mambu.loans.shared.model.LoanProduct;
+import com.mambu.organization.shared.model.Branch;
+import com.mambu.organization.shared.model.Centre;
+import com.mambu.savings.shared.model.SavingsAccount;
+import com.mambu.savings.shared.model.SavingsProduct;
+
+/**
+ * MambuEntity class is an enum for Mambu entities supported by the API wrapper library. This class is used to specify
+ * Mambu entity for an API operation
+ * 
+ * @author mdanilkis
+ * 
+ */
+public enum MambuEntity {
```

Comment:
Could you make more explicit what you mean by "Mambu class"? E.g. describe that the Mambu model class is meant here?